slick-codegen/src/main/scala/slick/codegen/AbstractSourceCodeGenerator.scala

```diff
@@ -59,39 +59,37 @@ abstract class AbstractSourceCodeGenerator(model: m.Model)
 
     //build and nest tuples if > 22 elements
     def tupleBuilder(values: Seq[String] , prefix: String): String = {
-      def mkTuple[T](l: Seq[T], _prefix: String) = l.map{ _prefix + _ }.mkString("(" , " ," , ")")
+      def mkTuple[T](l: Seq[T], _prefix: String) = l.map{ _prefix + _ }.mkString("(" , ", " , ")")
       if(values.size <= 22) 
-        mkTuple(values,"") 
+        mkTuple( values, "" )
       else 
-        mkTuple( values.grouped(22).toList.map {mkTuple(_,prefix)} , ""  )
+        mkTuple( values.grouped(22).toList.map {mkTuple(_,prefix)}, "" )
     }
 
     def tupleFactoryBuilder(columns: Seq[Column]): String = {
-        if(columns.size <= 22 ) s"${TableClass.elementType}.tupled"
```

Comment:
no need to have this in here. I think it is clearer to have it in the call site instead